core/tasks.py

```
@@ -256,3 +269,200 @@ def track_state_change(
+    """
+    try:
+        blog_post = GeneratedBlogPost.objects.get(id=blog_post_id)
+        
+        # Submit the blog post
+        success = blog_post.submit_blog_post_to_endpoint()
+        
+        if success:
+            blog_post.posted = True
+            blog_post.save(update_fields=['posted'])
+            
+            result = f"Successfully submitted blog post '{blog_post.post_title}' for {blog_post.project.name}"
```

Ruff (0.11.9)

Line too long (110 > 100)

(E501)